packages/slice-machine/components/CustomTypeTable/ctPage.tsx

```diff
@@ -0,0 +1,64 @@
+import { StatusBadgeWithTooltip } from "@components/Badge/StatusBadgeWithTooltip";
+import { CustomTypeSM } from "@slicemachine/core/build/models/CustomType";
+import Link from "next/link";
+import React from "react";
+import { Box, Text } from "theme-ui";
+
+export const CustomTypeTable: React.FC<{
+  customTypes: CustomTypeSM[];
+}> = ({ customTypes }) => {
+  const firstColumnWidth = "33%";
+  const secondColumnWidth = "33%";
+  const thirdColumnWidth = "17%";
+  const fourthColumnWidth = "17%";
+
+  return (
+    <Box
+      as={"table"}
+      sx={{
+        mt: "36px",
+      }}
+    >
+      <thead>
+        <tr>
+          <Box as={"th"} sx={{ width: firstColumnWidth }}>
+            Name
+          </Box>
+          <Box as={"th"} sx={{ width: secondColumnWidth }}>
+            API ID
+          </Box>
+          <Box as={"th"} sx={{ width: thirdColumnWidth }}>
+            Type
+          </Box>
+          <Box as={"th"} sx={{ width: fourthColumnWidth }}>
+            Status
+          </Box>
+        </tr>
+      </thead>
+      <tbody>
+        {customTypes.map((customType) => (
+          <Link passHref href={`/cts/${customType.id}`} key={customType.id}>
```

Comment:
so if they click any where on or around the row, it's a link?
...wait, so it's passes the href to the children, like this https://nextjs.org/docs/api-reference/next/link#if-the-child-is-a-custom-component-that-wraps-an-a-tag

Reply:
should we add tabIndex? or a tables accessable by default?

Reply:
yes clicking will send you to the CT page. the <Link/> code was already there I just moved it to it's own component

Reply:
ohh good point! I never think of accessibility.
No they were not accessable by default, added a tabIndex :)